### PR TITLE
feat: port RichSelectTags

### PR DESCRIPTION
**Pull request by chambo-e**
## Type

- Migration

### Summarise concisely:

I kinda struggled to have 100% coverage, I don't know why it does not want to include the deletion while the test is executed

#### What is expected?

Screenshot 2021-05-09 at 01 35 56
Screenshot 2021-05-09 at 01 35 59

### CODE REVIEW

**Review by vincentaudebert — APPROVED** [review]

File: src/components/RichSelectTags/index.js [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+        newPotentielTag: e,
+      })
+      onChangeSelectValue(e)
+    }
+  }
+
+  const onChangeTag = e => {
+    if (e) {
+      setTags({ newPotentielTag, stateTags: e })
+      onChange(e)
```
Comment (matthprost) [inline src/components/RichSelectTags/index.js:71]: In your tests onChange has empty paramters, tests are not passing trough those 2 lines

File: src/components/RichSelectTags/__stories__/index.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+<Meta title="Components/RichSelectTags" />
+
+# RichSelectTags
+
+## Controlled
+
+RichSelectTags only work as [controlled component](https://reactjs.org/docs/forms.html).
+
+<Canvas>
+  <Story name="basic">
```
Comment (matthprost) [inline src/components/RichSelectTags/__stories__/index.stories.mdx:14]:
```suggestion
  <Story name="Basic">
```

File: src/components/RichSelectTags/index.js [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+            {ctaText}
+          </Button>
+        </Tooltip>
+      </RichSelectContainer>
+      <Tags
+        manualInput={false}
+        tags={stateTags}
+        areTagsObject
+        onChange={onChangeTag}
+        variant="bordered"
```
Comment (matthprost) [inline src/components/RichSelectTags/index.js:120]: `RichSelect` and `Tags` have name as required prop, you can add a name to propTypes of `RichSelectTags` as required and add it in both components with prefix such as `name={"richselect-${name}"}` and `name={"tags-${name}"}`

File: src/components/RichSelectTags/index.js [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+      />
+    </>
+  )
+}
+
+RichSelectTags.propTypes = {
+  ctaText: PropTypes.string.isRequired,
+  ctaDisabled: PropTypes.bool,
+  ctaTooltip: PropTypes.string,
+  placeholder: PropTypes.string.isRequired,
```
Comment (matthprost) [inline src/components/RichSelectTags/index.js:130]: Is there a reason to put placeholder as required ? Maybe we could give it a default value

File: src/components/RichSelectTags/index.js [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+        areTagsObject
+        onChange={onChangeTag}
+        variant="bordered"
+      />
+    </>
+  )
+}
+
+RichSelectTags.propTypes = {
+  ctaText: PropTypes.string.isRequired,
```
Comment (matthprost) [inline src/components/RichSelectTags/index.js:127]: Same for this prop, it could have a default value I think

**Review by matthprost — CHANGES_REQUESTED** [review]

File: src/components/RichSelectTags/index.js (hunk 1 above)
Comment (matthprost) [reply src/components/RichSelectTags/index.js:72]: mmh that's strange 🤔

File: src/components/RichSelectTags/__stories__/index.stories.mdx [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+<Meta title="Components/RichSelectTags" />
+
+# RichSelectTags
+
+## Controlled
+
+RichSelectTags only work as [controlled component](https://reactjs.org/docs/forms.html).
+
+<Canvas>
+  <Story name="Basic">
```
Comment (matthprost) [inline src/components/RichSelectTags/__stories__/index.stories.mdx:14]:
```suggestion
  <Story name="Basic" height="300px">
```

will avoid options hide under story:

Screenshot 2021-05-11 at 14 04 50

**Review by matthprost — CHANGES_REQUESTED** [review]

**Review by matthprost — APPROVED** [review]

**Pull request merged**